### PR TITLE
Add topics as a column

### LINKED ISSUE #225: x.concepts is soon to be deprecated
From the [docs](https://docs.openalex.org/api-entities/authors/author-object#x_concepts):

> x_concepts will be deprecated and removed soon. We will be replacing this functionality with [Topics](https://docs.openalex.org/api-entities/topics) instead.

### PR DESCRIPTION
**Pull request by trangdata**
Closes #225 

- For Works: topics is now a column in addition to concepts
- For other entities, "topics" column replaces "concepts" column

https://docs.openalex.org/api-entities/concepts:
> The original OpenAlex Concepts are being deprecated in favor of [Topics](https://docs.openalex.org/api-entities/topics). We will continue to provide these Concepts for Works, but we will not be actively maintaining, updating, or providing support for these concepts. Unless you have a good reason to be relying on them, we encourage you to look into [Topics](https://docs.openalex.org/api-entities/topics) instead.

### CODE REVIEW

File: R/simplify.R [hunk 1]
```diff
@@ -113,6 +112,9 @@ show_works <- function(x, simp_func = utils::head) {
 }
 
 get_auth_position <- function(y, position = "first") {
+  if (identical(y, NA)) {
```
Comment (yjunechoe) [inline R/simplify.R:115]: Naive Q from not having explored the possible inputs to this function much: is there something specific to logical NA values for `y` such that we're targeting it with `identical()`? Could we use `is.na()` here to broadly coerce any type of NA into `NA_character_`? I wouldn't mind redundantly having `NA_character_` be a target of this condition (probably just me overinterpreting NAs though 😅)

**Review by yjunechoe — APPROVED** [review]
LGTM!

File: R/simplify.R (hunk 1 above)
Comment (trangdata) [reply R/simplify.R:115]: I was not sure of a good way to do this, but y can be a vector, and so is.na(y) can have more than one boolean value. I can do `if (length(y) == 1) && is.na(y))`. Perhaps that's better? I'm not sure I get your 2nd point of having `NA_character_` be a target of this condition...

Comment (yjunechoe) [reply R/simplify.R:115]: Ah! Got it. And sorry I didn't express this clearly: I mistakenly thought the intent of `identical(y, NA)` was to match logical NA but not match character (or some other) NA, so I took that as special casing logical NA specifically (again, this is probably just me reading too deep into different NA values!).

The only risk of `identical(y, NA)` is if we somehow receive e.g., `NA_character_`, it'd return `FALSE`. If that's a weird thing to worry about given the way `get_auth_position()` is called internally, please ignore this noise!!

**Pull request merged**